src/helpers.ml

```diff
@@ -86,11 +86,16 @@ let unmarshal_int32 s = Int64.to_int32 (unmarshal_int 4 s)
 let unmarshal_int64 s = unmarshal_int 8 s
 
 let unmask mask str =
-  if String.length str > 0 then begin
+  if String.length str = 0 then str
+  else
```

Comment:
I think it's worth adding `begin`/`end` here for more clarity

Reply:
Ok